idl/response/foxsi_get_shutter_transmission.pro

```diff
@@ -24,7 +25,8 @@
 ; EXAMPLES : None
 ;
 
-FUNCTION foxsi_get_shutter_transmission, shutter_state, ENERGY_ARR = energy_arr, PLOT = plot
+FUNCTION foxsi_get_shutter_transmission, shutter_state, SHUTTER_THICK_MM = shutter_thick_mm, $
+				 ENERGY_ARR = energy_arr, PLOT = plot, _EXTRA =_extra

```

Comment:
On the flip side, here you are accepting the special `_extra` keyword, but you aren't passing it on to be used by anything.